dist/index.js

```diff
@@ -3784,6 +3784,43 @@ exports.request = request;
 //# sourceMappingURL=index.js.map
 
 
+/***/ }),
+
+/***/ 2183:
+/***/ ((module, __unused_webpack_exports, __nccwpck_require__) => {
+
+"use strict";
+
+
+const core = __nccwpck_require__(2186)
+const github = __nccwpck_require__(5438)
+
+/**
+ * Displays warning message if the action reference is pinned to master/main
+ */
+function logActionRefWarning() {
+  const actionRef = process.env.GITHUB_ACTION_REF
+  const repoName = github.context.payload.repository.full_name
+
+  if (actionRef === 'main' || actionRef === 'master') {
+    core.warning(
+      `${repoName} is pinned at HEAD. We strongly ` +
+        `advise against pinning to "@master" as it may be unstable. Please ` +
```

Comment:
I know it is the dist, but should not we provide the right message?

```suggestion
        `advise against pinning to "@${actionRef}" as it may be unstable. Please ` +
```